presto-main/src/main/java/com/facebook/presto/sql/analyzer/AggregationAnalyzer.java

```diff
@@ -306,6 +307,18 @@ protected Boolean visitInPredicate(InPredicate node, Void context)
         protected Boolean visitFunctionCall(FunctionCall node, Void context)
         {
             if (metadata.isAggregationFunction(node.getName())) {
+                if (node.getSortItems().isPresent()) {
+                    for (SortItem sortItem : node.getSortItems().get()) {
+                        Expression sortKey = sortItem.getSortKey();
```

Comment:
Seems like the current error message in verifyNoOrderByReferencesToOutputColumns could apply to both function arguments and order by arguments?